WordPressAuthenticator/NUX/NUXButtonViewController.swift

```diff
@@ -34,13 +34,23 @@ open class NUXButtonViewController: UIViewController {
 
     // MARK: - Properties
 
-    @IBOutlet private var shadowView: UIImageView?
     @IBOutlet var stackView: UIStackView?
     @IBOutlet var bottomButton: NUXButton?
     @IBOutlet var topButton: NUXButton?
     @IBOutlet var tertiaryButton: NUXButton?
     @IBOutlet var buttonHolder: UIView?
 
+    @IBOutlet private var shadowView: UIImageView?
+    @IBOutlet private var shadowViewEdgeConstraints: [NSLayoutConstraint]!
```

Comment:
is there a reason to implicitly unwrap this one only?

Reply:
Hmm, this was the default (created by ctrl+dragging from Interface Builder), whereas the other optional property already existed. It looks like shadowView should always exist, so I could make that implict too if you like? Or I could make the constraints optional?

Reply:
I guess it's ok to leave it as it is, it just popped my eye but yeah IB always defaults to implicit.